project/Settings.scala

```diff
@@ -43,18 +43,19 @@ object Settings {
     val scalaZStream = "0.8.6a"
 
     // Scala libraries
-    val http4s       = "0.15.2a"
-    val squants      = "1.0.0"
+    val http4s       = "0.15.6a"
```

Comment:
Not really. 0.16.0 is a bigger but it uses cats-fs2 so we'll have to see when do we migrate